Unable to create stable release

/kind bug

**What steps did you take and what happened:**
```
csctl version
csctl version: v0.0.3
commit: 19b87a074800a411922cd49a89c45b50ab718e8f
```
```
cd /tmp
git clone https://github.com/SovereignCloudStack/cluster-stacks
cd cluster-stacks
git switch release-1.28
export GIT_PROVIDER=github
export GIT_ORG_NAME=SovereignCloudStack
export GIT_REPOSITORY_NAME=cluster-stacks
export GIT_ACCESS_TOKEN=<GH PAT> // was tested for validity
csctl create ~/workspace/scs/cluster-stacks/providers/openstack/scs -o .release
path "/home/janjan/workspace/scs/cluster-stacks/providers/openstack/scs": cluster stack hash: "y6ddwdbbwfb2nq959qzlvfvhvebswdc32ltsxfcgq4"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x219da04]

goroutine 1 [running]:
github.com/SovereignCloudStack/csctl/pkg/cmd.GetCreateOptions({0x28e2418, 0x387bde0}, {0x7ffeb4c5fcc4, 0x41})
	/home/janjan/workspace/scs/csctl/pkg/cmd/create.go:137 +0x6a4
github.com/SovereignCloudStack/csctl/pkg/cmd.createAction(0xc000110700?, {0xc0005743f0?, 0x4?, 0x25e3aa6?})
	/home/janjan/workspace/scs/csctl/pkg/cmd/create.go:202 +0x137
github.com/spf13/cobra.(*Command).execute(0x372f100, {0xc000574390, 0x3, 0x3})
	/home/janjan/workspace/scs/csctl/vendor/github.com/spf13/cobra/command.go:983 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0x372f3e0)
	/home/janjan/workspace/scs/csctl/vendor/github.com/spf13/cobra/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/janjan/workspace/scs/csctl/vendor/github.com/spf13/cobra/command.go:1039
github.com/SovereignCloudStack/csctl/pkg/cmd.Execute()
	/home/janjan/workspace/scs/csctl/pkg/cmd/root.go:37 +0x1a
main.main()
	/home/janjan/workspace/scs/csctl/main.go:23 +0xf
```

**What did you expect to happen:**
Same as with hash mode, since that works:
```
csctl create /tmp/cluster-stacks/providers/openstack/scs -o .release -m hash
path "/tmp/cluster-stacks/providers/openstack/scs": cluster stack hash: "y6ddwdbbwfb2nq959qzlvfvhvebswdc32ltsxfcgq4"
Creating output in .release/openstack-scs-1-28-v0-sha-y6ddwdb
.tmp/cluster-addon/Chart.yaml updating version from v1 to v0-sha.y6ddwdb
.tmp/cluster-class/Chart.yaml updating version from v1 to v0-sha.y6ddwdb
path now: ".tmp/cluster-class"
path now: ".tmp/cluster-addon"
Calling Provider Plugin: /home/janjan/.local/bin/csctl-openstack
config.yaml copied to releaseDir as node-images.yaml successfully!
Created .release/openstack-scs-1-28-v0-sha-y6ddwdb
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- csctl version: (use `csctl version`)
- OS (e.g. from `/etc/os-release`):

--- a/pkg/cmd/create.go
+++ b/pkg/cmd/create.go
@@ -128,6 +128,8 @@ func GetCreateOptions(ctx context.Context, clusterStackPath string) (*CreateOpti
 	case hashMode:
 		createOption.Metadata = clusterstack.HandleHashMode(createOption.CurrentReleaseHash, config.Config.KubernetesVersion)
 	case stableMode:
+		createOption.Metadata = &clusterstack.MetaData{}
+
 		gc, err := client.NewFactory().NewClient(ctx)
 		if err != nil {
 			return nil, fmt.Errorf("failed to create new github client: %w", err)

